File: src/Java/Store/4. Third Hole/src/main/refactoringgolf/store/ImageInfo.java
package refactoringgolf.store;

public class ImageInfo {

	public String path;

	public ImageInfo(String path) {
		this.path = path;
	}

	public String getPath() {
		return path;
	}

	public String getImageType() {
		return path.substring(path.indexOf(".") + 1);

	}
}

File: src/Net/1. First Course/2. First Hole/RefactoringGolf.Store/ImageInfo.cs
namespace RefactoringGolf.Stack
{
    public class ImageInfo
    {
        public string Path { get; private set; }

        public ImageInfo(string path)
        {
            Path = path;
        }

        public string ImageType
        {
            get
            {
                return Path.Substring(Path.IndexOf(".") + 1);
            }
        }
    }
}